core/src/util/builders.cpp

```diff
@@ -4,9 +4,32 @@
 #include "rectangle.h"
 #include "geom.h"
 
+#include <memory>
+
 static auto& NO_TEXCOORDS = *(new std::vector<glm::vec2>); // denotes that texture coordinates should not be used
 static auto& NO_SCALING_VECS = *(new std::vector<glm::vec2>); // denotes that scaling vectors should not be used
 
+void* alloc(void* _userData, unsigned int _size) {
+    return malloc(_size);
+}
+
+void* realloc(void* _userData, void* _ptr, unsigned int _size) {
+    return realloc(_ptr, _size);
```

Comment:
The tessAlloc struct also allows passing nullptr for the realloc function, which tells libtess2 to not use realloc. I tried that in the configuration but it didn't make a measurable difference in my tests, so I left realloc in.